### PR TITLE
PRTL-1928. Fix/facets dont reset

### PR DESCRIPTION
**Pull request by anncatton**

### CODE REVIEW

**Comment by alex-wilmer** [comment]
@cy CaseFacetSelection does use relay, it's wrapped in a Query. It probably should be generalized but I suggested to do this just for one index type to get started with.

Once we're sure this PR didn't break anything, we should use it as a guide for the other indices.

**Comment by cy** [comment]
OK but actually is it necessary to change custom facets to fix the reset? Looking at prod, the explore case aggregation doesn't have the issue & it's using the old code. If the intention is to move to relay modern for the custom facets then sure, but let's not merge this until the other indices are done and end up in a weird in-between-state on `next`

**Comment by alex-wilmer** [comment]
Sure sounds good

File: src/packages/@ncigdc/modern_components/FileAggregations/FileAggregations.js [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+        }
+        relay={props.relay}
+        style={{ borderBottom: `1px solid ${props.theme.greyScale5}` }}
+        additionalProps={facet.additionalProps}
+      />
+    ))}
+  </div>
+);
+
+// export const FileAggregationsQuery = {
```
Comment (cy) [inline src/packages/@ncigdc/modern_components/FileAggregations/FileAggregations.js]: ✂️

**Pull request merged**